src/core/MIGRATION.md

```diff
@@ -855,8 +857,37 @@ import { npStart: { core } } from 'ui/new_platform';
 
 _See also: [Public's CoreStart API Docs](/docs/development/core/public/kibana-plugin-public.corestart.md)_
 
+##### Application Services
+In client code, we have a series of shared application services that are being built in the shape of the new platform, but still technically reside in the legacy world. To expose these "shim plugin" contracts for you to consume in your own plugin's shim, we are creating dedicated exports for the `setup` and `start` contracts, which export APIs in the shape you will receive them in the new platform.
```

Comment:
Can we make it more clear that these are plugins and not services provided by default?

Reply:
Yep, I will circle back to update this after we have some follow up discussions today & tomorrow